tests/unit/lms/services/course_test.py

```diff
@@ -9,6 +9,7 @@
 
 
 class TestCourseService:
+    @pytest.mark.usefixtures("application_instance")
```

Comment:
This `application_instance` fixture was previously `autouse`. Only a few tests need it now so I removed the `autouse` and added explicitly to the tests that actually need it